Modules/CaPTkCommon/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ include_directories(
 )
 
 mitk_create_module(
-  #INCLUDE_DIRS 
+  INCLUDE_DIRS include src/internal
```

Comment:
Modules do not have an ~/internal folder

Reply:
I'm surprised how you managed to review this, since it was already changed before you reviewed. Maybe you didn't refresh your browser.

Anycase, this has been resolved and there is no internal/ anymore

Reply:
ok